internal/command/views/show_test.go

```diff
@@ -53,7 +53,7 @@ func TestShowHuman(t *testing.T) {
 			},
 			testSchemas(),
 			true,
-			"\n",
+			"The state file is empty. No resources are represented.\n",
```

Comment:
This seems like a good change in behaviour, and our human-readable output is largely exempt from backwards compatibility guarantees, so I think it's fine. We should probably note it in the changelog, though.

Reply:
I'll update the changelog 👍